injitest/src/main/java/inji/pages/ReceiveCardPage.java

```diff
@@ -32,6 +35,13 @@ public void clickOnAllowButton() {
             clickOnElement(allowButton);
         }
     }
+    
+    public ReceiveCardPage acceptPermissionPopupForLocation() {
```

Comment:
acceptPermissionPopupForLocation => acceptPermissionPopupForLocationAndroid
if its only for android